src/utils/clmfates_interfaceMod.F90

```diff
@@ -1716,7 +1716,7 @@ subroutine restart( this, bounds_proc, ncid, flag, waterdiagnosticbulk_inst, &
 
                   call ed_update_site( this%fates(nc)%sites(s), &
                         this%fates(nc)%bc_in(s), &
-                        this%fates(nc)%bc_out(s) )
+                        this%fates(nc)%bc_out(s),.true. )
```

Comment:
Same as above.

Reply:
Since this the one that uses .true. it would be good to have a comment that explains why this is true and the others are not.